pkg/plugins/golang/v2/webhook.go

```diff
@@ -90,6 +90,11 @@ func (p *createWebhookSubcommand) InjectResource(res *resource.Resource) error {
 		return err
 	}
 
+	alreadyHasAPI := p.config.HasResource(res.GVK)
+	if !alreadyHasAPI {
+		return fmt.Errorf("%s create webhook requires scaffolded API", p.commandName)
+	}
+
```

Comment:
This only works with `PROJECT` version 2, while go/v2 plugin may also use `PROJECT` version 3.